controllers/label_event_handler.go

```diff
@@ -26,3 +26,23 @@ func MapToOwnerByLabel(namespace, label string) func(object client.Object) []rec
 		return []reconcile.Request{}
 	}
 }
+
+// MapToOwnerByLabelValue returns an event handler will convert events on a resource to requests
+// if the resource matches a given label key and value
+// (if the label exists)
+func MapToOwnerByLabelValue(namespace, labelKey, labelValue string) func(object client.Object) []reconcile.Request {
```

Comment:
This doesn't map to any owner since the returned `Request` contains the name of the object.
I would maybe rename the func to `MapToControllerNsWhenMatchingLabel` or something like that.

Reply:
makes sense, renamed in https://github.com/codeready-toolchain/toolchain-common/pull/373/commits/28cde680248bb0135cbec44ad1e98ab9813605e9 . TBH I avoided the `Ns` part, but let me know WDYT. 

Thanks!

Reply:
Looks good :+1: